docs/changelog/111809.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 111809
+summary: Adding Field caps support for Semantic Text
+area: Relevance
```

Comment:
"Mapping" is probably a more appropriate area for this change

Reply:
(Note: This is auto-generated by labels in GitHub, may make sense to apply the label to direct to the Search/Mappings team as well)